tasks/main.yml

```diff
@@ -79,10 +79,22 @@
     mode: 0770
   become: "{{ deploydrupal_files_become }}"
 
+# See drush cache clear below for additional details.
+- name: clear caches.
+  shell: "{{ deploydrupal_drush_path }} cache-rebuild"
+  args:
+    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
+  when:
+    - deploydrupal_drush_cache_rebuild
```

Comment:
Instead of adding a second cache-rebuild here with a conditional, why don't we just swap the original order of `cache-rebuild` and `cache-clear drush`?